tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/DynamoDb/DeleteItemIntegration.cs

```diff
@@ -48,6 +50,22 @@ internal static CallTargetState OnMethodBegin<TTarget, TDeleteItemRequest>(TTarg
             var scope = AwsDynamoDbCommon.CreateScope(Tracer.Instance, Operation, out AwsDynamoDbTags tags);
             AwsDynamoDbCommon.TagTableNameAndResourceName(request.TableName, tags, scope);
 
+            if (!Tracer.Instance.Settings.SpanPointersEnabled)
+            {
+                return new CallTargetState(scope);
+            }
+
+            var tableName = request.TableName;
+            try
+            {
+                var keys = request.Keys.DuckCast<IDynamoDbKeysObject>();
+                SpanPointers.AddDynamoDbSpanPointer(scope.Span, tableName, keys);
+            }
+            catch (Exception exception)
+            {
+                Log.Debug("Unable to add span pointer: " + exception.Message);
```

Comment:
Nitpicky, but for readability/maintenability's sake it might be better to use String Interpolation:

```suggestion
                Log.Debug($"Unable to add span pointer: {exception.Message}");
```

I noticed it's also used a couple of times across the codebase

Reply:
**Edit:** I noticed after reading your code further that you used string interpolation later, too, so maybe you did this on purpose. If that's the case, please ignore me 😄

I just don't see why we wouldn't do it here as well

Reply:
Absolutely! you're right